connectors/src/lib/data_sources.ts

```diff
@@ -523,12 +523,12 @@ export async function upsertTableFromCsv({
         config: { ...e.config, data: undefined },
       };
       localLogger.error(
-        { error: sanitizedError },
-        "Error uploading structured data to Dust."
+        { error: sanitizedError, payload: dustRequestPayload },
```

Comment:
Will that be too much logging??

Reply:
Adding the payload here will dump the whole CSV on datadog. It could be fine while debugging, but let's remove this very quick once you have the log you need.

Reply:
Sure! Adding a note to myself